tests/services/generateFlightsByDate.test.ts

```diff
@@ -0,0 +1,27 @@
+import 'jest';
+import { DateTime } from 'luxon';
+import { generateFlightsByDate } from '../../src/services/generateFlightsByDate';
+
+describe('FlightCache ', () => {
+  const dateString = '2021-01-01';
+  const isoDate = DateTime.fromISO(dateString, { zone: 'utc' });
+
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  it('should generate flights by seed', () => {
+    const flights = generateFlightsByDate(isoDate);
+    expect(flights.length).toBeGreaterThan(0);
+  });
+
+  it('should generate flights from cache when same seed is used', () => {
+    const flights1 = generateFlightsByDate(isoDate);
+    expect(flights1.length).toBeGreaterThan(0);
+
+    const flights2 = generateFlightsByDate(isoDate);
+    expect(flights2.length).toBeGreaterThan(0);
+
+    expect(flights1).toEqual(flights2);
+  });
+});
```

Comment:
A couple quick changes here, and a test update.

The flight cache test is currently testing if the same values are being returned, but we need to make sure that the second time is coming from the cache. I would mock the `FlightCache` to make sure that `cacheFlights` was called the first time, and then not called the second time by making sure it was only called once. That would make sure that it's not generating the flights again the second time.

```suggestion
describe('generateFlightsByDate ', () => {
  const dateString = '2021-01-01';
  const isoDate = DateTime.fromISO(dateString, { zone: 'utc' });

  it('should generate flights by seed', () => {
    const flights = generateFlightsByDate(isoDate);
    expect(flights.length).toBeGreaterThan(0);
  });

  it('should return flights from cache when same date is used', () => {
    const flights1 = generateFlightsByDate(isoDate);
    expect(flights1.length).toBeGreaterThan(0);
    // expect(cacheFlightsMock).toHaveBeenCalledWith(........);

    const flights2 = generateFlightsByDate(isoDate);
    expect(flights2.length).toBeGreaterThan(0);

    expect(flights1).toEqual(flights2);
    // expect(cacheFlightsMock).toHaveBeenCalledTimes(1);
  });
});
```